tests/unit/woopay/test-class-woopay-session.php

```diff
@@ -204,6 +199,6 @@ private function setup_adapted_extensions() {
 	}
 
 	private function set_is_woopay_eligible( $is_woopay_eligible ) {
-		$this->mock_cache->method( 'get' )->willReturn( [ 'platform_checkout_eligible' => $is_woopay_eligible ] );
+		update_option( 'wcpay_account_data', [ 'data' => [ 'platform_checkout_eligible' => $is_woopay_eligible ] ] );
```

Comment:
If we go back to using the WooPayments classes I think we need to revert these tests?